enhancements/sdk-generate-commands.md

```diff
@@ -0,0 +1,353 @@
+---
+title: README describing new command workflows for OLM enabled operators
```

Comment:
Nit: I did not get the title. maybe something as;
`OLM command workflows with Kubebuider integration`
`OLM command workflows with new layout`

Reply:
Actually if this is truly to follow the template and openshift specs, the title should be filename minus the `.md` i.e. it should be `sdk-generate-commands`.

https://github.com/operator-framework/enhancements/blob/master/enhancements/template.md#title

> The title should be lowercased and spaces/punctuation should be replaced with -.